src/libraries/Microsoft.PowerFx.Core/Public/Values/DateTimeValue.cs

```diff
@@ -28,7 +28,10 @@ public override void Visit(IValueVisitor visitor)
 
         public override void ToExpression(StringBuilder sb, FormulaValueSerializerSettings settings)
         {
-            sb.Append($"DateTimeValue({CharacterUtils.ToPlainText(Value.ToString("o", CultureInfo.InvariantCulture))})");
+            var date = Value.Date;
+            var timeSpan = Value.TimeOfDay;
```

Comment:
No need to break it down - we can access the values (Year/Month/Day/Hour/Minute/Second/Millisecond) from the DateTime instance directly